debezium-connector-mongodb/src/test/resources/Dockerfile.rest.test

```diff
@@ -0,0 +1,29 @@
+ARG BASE_IMAGE
+ARG DEBEZIUM_VERSION
+FROM ${BASE_IMAGE}
+ARG DEBEZIUM_VERSION
+
+ENV CONNECTOR="mongodb"
+
+RUN echo "Installing Debezium connectors version: ${DEBEZIUM_VERSION}" ; \
+MAVEN_REPOSITORY="https://repo1.maven.org/maven2/io/debezium" ; \
+if [[ "${DEBEZIUM_VERSION}" == *-SNAPSHOT ]] ; then \
+    MAVEN_REPOSITORY="https://oss.sonatype.org/content/repositories/snapshots/io/debezium" ; \
+fi ; \
+CONNECTOR_VERSION="${DEBEZIUM_VERSION}" ; \
+for PACKAGE in {scripting,}; do \
+    local CONNECTOR_VERSION="${DEBEZIUM_VERSION}" ; \
+    if [[ "${DEBEZIUM_VERSION}" == *-SNAPSHOT ]] ; then \
+        CONNECTOR_VERSION=$(curl --silent -fSL "${MAVEN_REPOSITORY}/debezium-${PACKAGE}/${DEBEZIUM_VERSION}/maven-metadata.xml" | awk -F'<[^>]+>' '/<extension>tar.gz<\/extension>/ {getline; print $2; exit}'); \
+    fi ; \
+    echo "Downloading and installing debezium-${PACKAGE}-${CONNECTOR_VERSION}.tar.gz ..." ; \
+    curl --silent -fSL -o /tmp/package.tar.gz "${MAVEN_REPOSITORY}/debezium-${PACKAGE}/${DEBEZIUM_VERSION}/debezium-${PACKAGE}-${CONNECTOR_VERSION}.tar.gz" && \
+    echo "Extracting debezium-${PACKAGE}-${CONNECTOR_VERSION}.tar.gz ..." && \
+    tar -xzf /tmp/package.tar.gz -C $EXTERNAL_LIBS_DIR && \
+    echo "Successfully installed debezium-${PACKAGE}-${CONNECTOR_VERSION}!" ; \
+    rm -f /tmp/package.tar.gz ; \
+done
+
+COPY --chown=kafka:kafka debezium-connector-${CONNECTOR}-${DEBEZIUM_VERSION}-plugin.tar.gz /tmp/plugin.tar.gz
+
+RUN tar -xvzf /tmp/plugin.tar.gz -C ${KAFKA_CONNECT_PLUGINS_DIR}/ ; rm -f /tmp/plugin.tar.gz
```

Comment:
Don't we have a support fro extending the KC container image in our TC wrapper already? I can't image anyone will keep these dockerfiles in sync.

Reply:
Are we using KC in such a way though? 

@Naros ?

Reply:
I don't agree that we have to accept anything. We should address these things now and not just accept that it is gonna be bad.

Reply:
@jcechace I think we can only influence runtime config but we can't re-configure the image structure / content. Especially what we can't do imo is using artifacts that we just built inside those containers. That's the gap I am filling here.
Please file a Jira if there's something that we should do or evaluate to come closer to the solution you are having in mind. Please attach it as sub-task to this parent task/epic: https://issues.redhat.com/browse/DBZ-4395